mailpoet/lib/WooCommerce/Tracker.php

```diff
@@ -39,6 +44,7 @@ public function addTrackingData($data): array {
       $analyticsData = $this->newslettersRepository->getAnalytics();
       $data['extensions']['mailpoet'] = [
         'campaigns_count' => $analyticsData['campaigns_count'],
+        'currency' => $this->wooHelper->getWoocommerceCurrency(),
```

Comment:
I think in the purchase statistics, we also include currencies. If a currency is changed or multiple are mixed for some reason, it will not match what will be tracked here. That said, maybe it's quite an edge case.

I'm not fully sure what we're doing in the UI, but I think maybe we consider only those states whose currency equals the currently set currency. That might be an option for a simple improvement, although I guess that may mean saving the currency per campaign or something.

Reply:
Good idea! I'll filter only revenues that are in the current shop currency.